.github/workflows/cli.yml

```diff
@@ -0,0 +1,53 @@
+name: "CLI: Build & Install"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x]
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: latest
+
+      - name: Install dependencies
+        working-directory: ./create-onchain
+        run: bun install
+
+      - name: Build CLI
+        working-directory: ./create-onchain
+        run: bun run build
+
+      - name: Make CLI executable
+        run: chmod +x ./create-onchain/dist/esm/cli.js
+
+      - name: Create test project
+        run: |
+          mkdir test-project
+          cd test-project
+          yes "" | ../create-onchain/dist/esm/cli.js
+        env:
+          CI: true
+
+      - name: List directories
+        run: ls -la test-project
```

Comment:
if the purpose here is to confirm that create-onchain worked correctly, should we turn an explicit test for a file that we'd expect? 

something like:
```
test -d test-project/my-onchainkit-app/app/page.tsx || exit 1
test "$(ls -A test-project/my-onchainkit-app)" || exit 1
```

Reply:
Sorry, that was for testing purposes only - will remove